packages/api/src/schema/api-schema.yaml

```diff
@@ -286,20 +286,36 @@ components:
       properties:
         playbackId:
           type: string
-          description: Playback ID of the stream or asset to clip
+          description: >-
+            The playback ID of the stream or stream recording to clip. Asset
+            playback IDs are not supported yet.
         startTime:
           type: number
-          description:
-            Start time of the clip in UNIX timestamp (in milliseconds)
+          description: >-
+            The start timestamp of the clip in Unix milliseconds. This is the
+            **server time** - not relative time to the start of the stream. _See
+            the ClipTrigger in the UI Kit for an example of how this is
+            calculated (for HLS, it uses `Program Date-Time` tags, and for
+            WebRTC, it uses the latency from server to client at stream
+            startup)._
         endTime:
           type: number
-          description: End time of the clip in UNIX timestamp (in milliseconds)
+          description: >-
+            The end timestamp of the clip in Unix milliseconds. This is the
+            **server time** - not relative time to the start of the stream. _See
```

Comment:
Does this description make sense for the end time? I'm thinking server term terminology will be confusing for most users. Should we just leave out the server time bit from both startTime and endTime?

Reply:
Yeah, I understand what you mean - I mean, it _is_ the server time though, right? It seems like it's hard to explain. If we say Unix time, doesn't that mean we're constantly explaining the latency for HLS vs WebRTC?

I can remove that verbiage though. Maybe this is resolved with a better interface and not documentation.

Reply:
I removed that verbiage and we can reconsider when we come back to clipping.